Examples/App.config

```diff
@@ -2,5 +2,5 @@
 <configuration>
     <startup> 
         
-    <supportedRuntime version="v2.0.50727"/></startup>
+    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.0,Profile=Client"/></startup>
```

Comment:
To integrate unit test, targeted .Net Framework 4 Client Profile.
Multi targeting would be useful to keep compatibility with .Net Framework 2.0 ~ 3.5.